backend/app/assets/javascripts/spree/backend/country_states.js

```diff
@@ -0,0 +1,85 @@
+// Attach a 'js-country-states' class to a wrapper around both fields
+// Attach a 'js-country' class to country select wrapper
+// Attach a 'js-state' class to the state select wrapper
+// Now the states will be refreshed as the country changes.
+
+$(document).ready(function () {
+  $('.js-country select').on('change', function () {
+    var $parent = $(this).parents('.js-country-states');
+    requestStates($parent);
+  });
+});
+
+function requestStates($parent){
```

Comment:
would it make sense to namespace these functions rather than putting them on the window?